expo/gbe/migrations/0008_auto__add_usermessage__add_unique_usermessage_view_code.py

```diff
@@ -0,0 +1,387 @@
+# -*- coding: utf-8 -*-
```

Comment:
Yes, and I'm honestly afraid to change it - there's something really weird happening with the gbe_costumes - I had to make a migration (the 0007 migration) to get this migration to work... but the I had to fake it or I couldn't run 0008 and then I had to delete it, or the unit tests wouldn't run.